packages/react-router/index.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useRef } from 'react';
```

Comment:
```suggestion
import React from 'react';
```

As per the previous nit pick.